app/pages/collections/list.cjsx

```diff
@@ -24,33 +26,72 @@ List = React.createClass
           '/simple-avatar.jpg'
 
   cardLink: (collection) ->
-    [collection_owner, collection_name] = collection.slug.split('/')
-    "/collections/#{collection_owner}/#{collection_name}"
+    baseLink = "/"
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}/"
+    "#{baseLink}collections/#{collection.slug}"
 
   listCollections: ->
```

Comment:
When viewing a [user's collections](http://localhost:3735/projects/alexbfree/computer-vision-serengeti/users/Whoandwhatitis/collections?env=production), this doesn't filter correctly.